### PR TITLE
Fixes #18707: Add documentation about set-force-audit in audit/enforce chapter

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/18707

### CODE REVIEW

**Comment by fanf** [comment]
PR updated with a new commit

File: src/reference/modules/usage/pages/configuration_management.adoc [hunk 1]
```diff
@@ -236,9 +236,10 @@ image:audit_mode_general_overview.png[]
 
 This mode can be set:
 
-* Globally on the Rudder root server. In this can case there are two options: allow to override this mode on specific items, or use the global configuration everywhere.
-* On a directive.
-* On a node.
+* Globally on the Rudder root server, in the `Settings > General` page. In this can case there are two options: allow to override this mode on specific items, or use the global configuration everywhere.
+* On a directive, in the directive configuration form, 
+* On a node, from the `Node details > Settings` page, 
+* from the node itself, which will override mode configured from Rudder server, thanks to `rudder set-force-audit` and `rudder unset-force-audit` commands.
```
Comment (amousset) [inline src/reference/modules/usage/pages/configuration_management.adoc:242]:
```suggestion
* From the node itself, which will override the mode configured on the Rudder server, thanks to `rudder agent set-force-audit` and `rudder agent unset-force-audit` commands.
```

**Comment by fanf** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**